src/main/java/org/example/leetcode/FirstBadVersion.java

```diff
@@ -0,0 +1,19 @@
+/* The isBadVersion API is defined in the parent class VersionControl.
+      boolean isBadVersion(int version); */
+
+public class Solution extends VersionControl {
```

Comment:
VersionControl class is missing in this PR

Reply:
there is no need to define VersionControl class in this question